dpnp/backend/custom_1arg_1type_tbl.hpp

```diff
@@ -39,6 +39,22 @@
 #error "MACRO_CUSTOM_1ARG_1TYPE_OP is not defined"
 #endif
 
+#ifdef _SECTION_DOCUMENTATION_GENERATION_
+
+#define MACRO_CUSTOM_1ARG_1TYPE_OP(__name__, __operation1__, __operation2__)                                            \
+    /** @ingroup BACKEND_API                                                                                         */ \
+    /** @brief Per element operation function __name__                                                               */ \
+    /**                                                                                                              */ \
+    /** Function "__name__" executes operator "__operation1__" over each element of the array                        */ \
```

Comment:
We could also mention `__operation2__` in the doc.
```suggestion
    /** Function "__name__" executes operator either "__operation1__" or "__operation2__" over each element of the array                        */ \
```